src/api/java/CMakeLists.txt

```diff
@@ -9,20 +9,59 @@
 ## directory for licensing information.
 ##
 
-# create a directory for the cvc package
-file(MAKE_DIRECTORY "${CMAKE_CURRENT_BINARY_DIR}/cvc")
+# copy cvc libraries to the resource directory
+file(GLOB CVC_LIBS "${CMAKE_INSTALL_PREFIX}/lib/*.so*")
+file(COPY ${CVC_LIBS} DESTINATION "${CMAKE_CURRENT_BINARY_DIR}/src/main/resources")
```

Comment:
Is there any library other than `libcvc4.so` that you want to include?

Reply:
No, but for some reason, `libcvc4.so` does not work, only `libcvc4.so.7` works.
Actually I have to account for Mac and Windows, but for now I am focused on Linux

Reply:
Again, this should be solved by just using `$<TARGET_FILE:cvc4>`

Reply:
Changed to use `$<TARGET_FILE:cvc4>`